src/main.c

```diff
@@ -814,7 +817,32 @@ int main(int argc, char *argv[]) {
 #endif
 
   // Process command line arguments
-  while((ch = getopt(argc, argv, "?Ab:B:c:C:DeE:Fi:l:nNp:OP:qrtT:U:vVx:")) != -1) {
+  struct option longopts[] = {
+    {"help",       no_argument,       NULL, '?'},
+    {"baud",       required_argument, NULL, 'b'},
+    {"bitclock",   required_argument, NULL, 'B'},
+    {"programmer", required_argument, NULL, 'c'},
+    {"config",     required_argument, NULL, 'C'},
+    {"noerase",    no_argument,       NULL, 'D'},
+    {"erase",      no_argument,       NULL, 'e'},
+    {"logfile",    required_argument, NULL, 'l'},
+    {"test",       no_argument,       NULL, 'n'},
```

Comment:
`-n` meant originally don't write to the AVR part for `-U` operations, where `-n` would write the file to stdout instead of the AVR memory. When other ways to write to the AVR were implemented (`-t`, `-T`, ...) there was no way to test these in a dryrun as it were until the `-c` programmer `dryrun` was implemented. I now wonder whether `--test` should be `--test-U` or rather `--test-update` to clarify the role of that option.